schemas/enums/language.json

```
@@ -51,6 +51,7 @@
         "Erg",
         "Erlang",
         "F#",
+        "FacilityServiceDefinition",
```

We can include spaces here, I think it'd be more correct to use:

```suggestion
        "Facility Service Definition",
```